ducktape/command_line/config.py

```diff
@@ -38,7 +38,7 @@ class ConsoleConfig(object):
     RESULTS_ROOT_DIRECTORY = "./results"
 
     SESSION_LOG_FORMATTER = '[%(levelname)s:%(asctime)s]: %(message)s'
-    TEST_LOG_FORMATTER = '[%(levelname)-5s - %(asctime)s - %(module)s - %(funcName)s - lineno:%(lineno)s]: %(message)s'
+    TEST_LOG_FORMATTER = '[%(levelname)-5s %(asctime)s - %(module)s:%(lineno)s %(funcName)s]: %(message)s\033[0m'
```

Comment:
Is this actually safe to just include here, is it? I ask because I've definitely seen cases where escape codes bleed through when used indiscriminately, e.g. they fill automated build logs with a lot of noise because the escape codes are not filtered out. Is there any guidance in the logging package about how to handle this, or does it already strip out escape codes when you're not connected to a terminal? Alternatively, I think we have some detection of whether we're in a terminal and the size of the terminal somewhere; maybe we can use that to decide whether it's safe to add this escape code or not?